.github/workflows/pr.yml

```diff
@@ -0,0 +1,27 @@
+name: Perform Python Format
+
+on: pull_request
+
+jobs:
+  black:
+    name: Lint with ESLint
```

Comment:
I think you just changed this by mistake, the previous name was correct

Reply:
Yeh undid too far. Working out different approach for using Black